backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/ConfirmationCodeController.java

```diff
@@ -0,0 +1,48 @@
+package ca.gov.dtsstn.cdcp.api.web.v1.controller;
+import java.util.List;
+import java.util.stream.StreamSupport;
+
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ca.gov.dtsstn.cdcp.api.service.ConfirmationCodeService;
+import ca.gov.dtsstn.cdcp.api.web.v1.model.ConfirmationCodeModel;
+import ca.gov.dtsstn.cdcp.api.web.v1.model.mapper.ConfirmationCodeModelAssembler;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.constraints.NotBlank;
+import ca.gov.dtsstn.cdcp.api.service.domain.ConfirmationCode;
+
+@Validated
+@RestController
+@RequestMapping({ "/api/v1" })
```

Comment:
This should be `/api/v1/confirmation-codes` so that the openapi spec is handled correctly.

Reply:
mapping changed